avocado/core/test.py

```diff
@@ -52,18 +52,22 @@
 COMMON_TMPDIR_NAME = 'AVOCADO_TESTS_COMMON_TMPDIR'
 
 
-class TestName(object):
+class TestID(object):
 
     """
-    Test name representation
+    Test ID construction and representation according to specification
+
+    This class wraps the representation of both Avocado's Test ID
+    specification and Avocado's Test Name, which is part of a Test ID.
     """
 
     def __init__(self, uid, name, variant=None, no_digits=None):
         """
-        Test name according to avocado specification
+        Constructs a TestID instance
 
         :param uid: unique test id (within the job)
-        :param name: test name (identifies the executed test)
+        :param name: test name, as returned by the Avocado test resolver
+                     (AKA as test loader)
         :param variant: variant id
```

Comment:
It's not the `variant id` that will be received in this param, but the variant itself. Not this commit fault, but might be this commit fix :)